polaris-icons/CONTRIBUTING.md

```diff
@@ -0,0 +1,204 @@
+# Polaris icons contribution guidelines
+
+This repository is the place to contribute changes to icons and <https://polaris-icons.shopify.com>.
+
+- [Questions and troubleshooting](#questions-and-troubleshooting)
+- [Code of conduct](#code-of-conduct)
+- [Icon sets and design guidelines](#icon-sets-and-design-guidelines)
+- [Proposing new icons](#proposing-new-icons)
+- [Proposing updates to existing icons](#proposing-updates-to-existing-icons)
+- [Exporting icons as SVG files](#exporting-icons-as-svg-files)
+- [Repository structure](#repository-structure)
+- [Development quick start](#development-quick-start)
+- [Running tests](#running-tests)
+- [Adding an icon to the repository](#adding-an-icon-to-the-repository)
+- [Deprecation guidelines](#deprecation-guidelines)
+  - [Deleting an icon](#deleting-an-icon)
+  - [Renaming an icon](#renaming-an-icon)
+- [Publishing the library](#publishing-the-library)
+- [Deploying the Polaris icon explorer website](#deploying-the-polaris-icon-explorer-website)
+
+## Questions and troubleshooting
+
+If you need help or have questions, ask us in the [#polaris](https://app.slack.com/client/T024G4BGQ/C4Y8N30KD) Slack channel.
+
+## Code of conduct
+
+We expect all participants to read our [code of conduct](./.github/CODE_OF_CONDUCT.md) to understand which actions are and aren’t tolerated.
+
+## Icon sets and design guidelines
+
+To learn about best practices for designing and using Polaris icons in your designs, read the [Icon design guidelines](https://polaris.shopify.com/design/icons) in Polaris.
+
+## Proposing new icons
+
+If you can’t find the icons you need, you can [propose new icons](https://github.com/Shopify/polaris-icons/issues/new?assignees=%40shopify%2Ficon-guild&labels=New%2CProposal&template=propose-new-icons.md&title=%5BProposal%5D+Add+%3Cicon+name%28s%29%3E) to be included in the collection.
```

Comment:
Might be worth doing some quick URL updates here just incase we forget to come back and fix them later.